src/global_mapper_ros/global_mapper_ros.h

```diff
@@ -17,38 +19,37 @@ namespace global_mapper {
 class GlobalMapperRos {
  public:
   explicit GlobalMapperRos(volatile std::sig_atomic_t* stop_signal_ptr);
-  ~GlobalMapperRos();
 
   void Run();
 
   volatile std::sig_atomic_t* stop_signal_ptr_;
 
  private:
-  void GetParams();
+  void GetParams(GlobalMapperParams& global_mapper_params);
```

Comment:
I'm still waiting for that rule to pay off... in the meantime 907746c34494c6aa97dd2cd00d03eccbabdda717.